packages/fetch/tests/index.test.js

```diff
@@ -22,7 +22,7 @@ it('returns a tuple for successful requests', async () => {
 
```

Comment:
@allanchau, could you please add an additional test here to make sure the Response object properties are being passed through properly.

Reply:
@smebberson I've added the test in now. The main properties are present, but some of the ones listed here are not. https://developer.mozilla.org/en-US/docs/Web/API/Response#properties

Reply:
@allanchau, yeah, no worries.